src/development/metrics.md

```diff
@@ -0,0 +1,153 @@
+# Metrics
+
+This page gather all the measurement metrics, so that we can easily refer to them in the future. It is broken into four parts:
+
+ - The measurements that are common to both verkle and binary trees
+ - The measurements that are specific to verkle trees
+ - The measurements that are specific to binary trees
+ - The measurements that are specific to state expiry
```

Comment:
- I'd remove or try to dedupe "The measurements" into the previous sentence.
- We usually use capitals for tree names (e.g., Verkle Trees)
- We still have pending even the chapter on State Expiry, so maybe I'd suggest removing the last bullet, or maybe add an extra "(Soon..)